Lib/test/test_httplib.py

```diff
@@ -344,6 +344,17 @@ def test_invalid_headers(self):
                 with self.assertRaisesRegex(ValueError, 'Invalid header'):
                     conn.putheader(name, value)
 
+    def test_headers_debuglevel(self):
+        body = b'HTTP/1.1 200 OK\r\nFirst: val\r\n: nval\r\nSecond: val\r\n\r\n'
```

Comment:
What `: nval` means?

I think it would be better to write this literal as multiple lines, splitting at newlines.
```python
body = (b'HTTP/1.1 200 OK\r\n'
        b'First: val\r\n'
        ...
```